maxim/tests/test_portkey.py

```
@@ -0,0 +1,342 @@
+            )
+            print(f"Async response: {response}")
+            
+            if response.choices[0].message.tool_calls:
+                for tool_call in response.choices[0].message.tool_calls:
+                    print(f"Async tool call: {tool_call.function.name}")
+                    print(f"Arguments: {tool_call.function.arguments}")
+                    
+        except Exception as e:
+            print(f"Async tool test error: {e}")
+
+    def test_tool_call_without_tools_parameter(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_tool_call_without_tools_parameter`

Add return type annotation: `None`

(ANN201)